python-autobahn/install.yaml

```diff
@@ -0,0 +1,2 @@
+- type: system
+  name: python-autobahn
```

Comment:
I don't think this is required by rosbridge.

Reply:
dep of rosbridge_server, https://github.com/RobotWebTools/rosbridge_suite/blob/ad63eb1f7a05d8d52470ac1364b033c74683bbbf/rosbridge_server/package.xml#L18